doc/perception/experiments/CLRerNet - lanedetection/Lanedetection_node.py

```
@@ -0,0 +1,377 @@
+        if self.img_cnt >= self.img_freq:
+            self.image = self.bridge.imgmsg_to_cv2(ImageMsg, "bgr8")
+            # self.image = self.preprocess_image(self.image)
+
+            lane_mask = self.detect_lanes(self.image, self.model)
+
+            if lane_mask is not None:
+                lanedetection_image = self.bridge.cv2_to_imgmsg(lane_mask, "bgr8")
+                self.lane_publisher.publish(lanedetection_image)
+            else:
+                # Only there to see if node is running, if lane_mask returns null -> model not working yet
+                no_img = np.zeros_like(self.image)  # Creates an empty image with the same dimensions
```

Ruff (0.8.2)

Undefined name `np`

(F821)